src/test/java/br/com/cibus/admin/AdminControllerTest.java

```diff
@@ -0,0 +1,41 @@
+package br.com.cibus.admin;
+
+import br.com.cibus.tipodecozinha.RestaurantesPorTipoDeCozinha;
+import br.com.cibus.tipodecozinha.TipoDeCozinhaRepository;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.util.List;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@WebMvcTest(AdminController.class)
+public class AdminControllerTest {
+
+    @MockBean
+    private TipoDeCozinhaRepository tipoDeCozinhaRepository;
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Test
+    void dashboard() throws Exception {
+        RestaurantesPorTipoDeCozinha arabe = mock(RestaurantesPorTipoDeCozinha.class);
+        RestaurantesPorTipoDeCozinha baiana = mock(RestaurantesPorTipoDeCozinha.class);
+        RestaurantesPorTipoDeCozinha mexicana = mock(RestaurantesPorTipoDeCozinha.class);
+
+        List<RestaurantesPorTipoDeCozinha> restaurantesPorTipoDeCozinha = List.of(arabe, baiana, mexicana);
+        when(tipoDeCozinhaRepository.contaRestaurantesPorTipoDeCozinha()).thenReturn(restaurantesPorTipoDeCozinha);
+
+        mockMvc.perform(get("/admin"))
+                .andExpect(status().isOk())
+                .andExpect(model().attribute("restaurantesPorTipoDeCozinha", restaurantesPorTipoDeCozinha))
```

Comment:
Lucas, acho que dá pra deixar mais robusto ainda usando a versão do `modelI().attribute(...)` que [recebe um Matcher do Hamcrest](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/test/web/servlet/result/ModelResultMatchers.html#attribute-java.lang.String-org.hamcrest.Matcher-).

Aí, daria pra fazer algo como:

```suggestion
                .andExpect(model().attribute("restaurantesPorTipoDeCozinha", contains(arabe, baiana, mexicana)))
```

O import seria:

```
import static org.hamcrest.Matchers.contains;
```

O que acha?

Reply:
Pelo menos para esse cenário de comparação dos itens da lista o próprio `model().attribute()` simples já resolve, fiz um teste trocando a ordem dos itens e ele identificou a diferença. Mas foi bom saber que o `model().attribute()` tem uma opção de passar um matcher, não conhecia.

Reply:
Opa, Lucas! Então tá justo!